cmd/check.go

```diff
@@ -96,6 +96,13 @@ var checkCmd = &cobra.Command{
 
 func init() {
 	RootCmd.AddCommand(checkCmd)
+
+	if runtime.GOOS == "windows" {
+		defaultRunnerConfig = os.Getenv("SYSTEMDRIVE") + 
+		                       "\\DCOS\\diagnostics\\config\\dcos-diagnostics-runner-config.json"
+	}
```

Comment:
maybe use a constant, similar to `defaultRunnerConfig` to define a default path for the config on windows machine?

Reply:
Good idea.

I will add that.